lib/ddtrace/ext/http.rb

```diff
@@ -2,7 +2,7 @@ module Datadog
   module Ext
     module HTTP
       BASE_URL = 'http.base_url'.freeze
-      ERROR_RANGE = 500...600
+      ERROR_RANGE = 400...500
```

Comment:
this shoudnt get changed
```suggestion
      ERROR_RANGE = 500...600
```